### PR TITLE
CRF UI/UX Changes - add tips for measuring

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
- Added learn more link for showing tips for measuring to the heart rate step.
- Added functionality to skip the summary review step if the result is not included.

Note: This provides a partial resolution to https://sagebionetworks.jira.com/browse/HS-30 but uses the existing screens designed for a resting heart rate. The messaging is the same, but instead it just shows a "Done" button. I had thought we decided that we were *not* going to include a "Redo" button but instead return the result back to the main view controller with the option that the app would handle how to manage a failed stair step test.

### CODE REVIEW

File: CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/Cardio_Stair_Step.json [hunk 1]
```diff
@@ -206,7 +206,7 @@
     {
         "identifier": "feedback",
         "type": "completion",
-        "resultTitle": "Your Recovery HR Score is estimated as",
+        "resultTitle": "Your estimate Recovery HR score is",
```
Comment (Erin-Mounts) [inline CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/Cardio_Stair_Step.json]:
```suggestion
        "resultTitle": "Your estimated Recovery HR score is",
```

**Review by Erin-Mounts — APPROVED** [review]
one nit, otherwise looks good

File: CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateStepViewController.swift [hunk 2]
```diff
@@ -113,23 +113,33 @@ public final class CRFHeartRateStepViewController: RSDActiveStepViewController,
...
+    private var _firstAppearance: Bool = true
+    
     public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
         (self.countdownDial as! RSDCountdownDial).innerColor = UIColor.clear
+        
+        if _firstAppearance {
+            // Make sure the learn more button is hidden
+            self.learnMoreButton?.isHidden = true
```
Comment (DanWebster) [inline CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateStepViewController.swift:125]: IMG_0694
IMG_0695

Is this the learn more button you mean here (this is after failing to get an HR after the step test) that should be hidden?

Comment (syoung-smallwisdom) [reply CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateStepViewController.swift:125]: @DanWebster With a clean build, this file is packaged.

**Pull request merged**